modules/kmm-setting-kernel-firmware-search-path.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="kmm-setting-kernel-firmware-search-path_{context}"]
+= Setting the kernel firmware search path
+
+The Linux kernel accepts the `firmware_class.path` parameter as a search path for firmware, as explained in link:https://www.kernel.org/doc/html/latest/driver-api/firmware/fw_search_path.html[Firmware search paths].
+
+KMM workers can set that value on nodes by writing to sysfs, before attempting to load kmods.
```

Comment:
@qbarrand Reword/clarify this.

Reply:
What information is missing here?

Reply:
Should this change to "KMM worker pods"?